webrender/src/renderer.rs

```diff
@@ -945,6 +1006,11 @@ impl<B: hal::Backend> GpuCacheTexture<B> {
     fn flush(&mut self, device: &mut Device<B>) -> usize {
         let texture = self.texture.as_ref().unwrap();
         match self.bus {
+            #[cfg(not(feature = "gleam"))]
+            GpuCacheBus::PMbuffer { ref mut mapped_ranges } => {
+                device.flush_mapped_ranges(mapped_ranges.drain(..));
+                return 0
```

Comment:
no need for `return` here

Reply:
Fixed in https://github.com/szeged/webrender/pull/297/commits/61242837ca2384876662a54d0bc03f7d9e4727c5